### DISCUSSION TITLE
Neoforge and forge compat in the future like after 1.21 maybe make forge and neoforge together like you have forge and fabric

### DISCUSSION THREAD

**Discussion by Thunderrock424242** [Sinytra/Connector#819] [Ideas]
.

**Comment by coco875** [comment]
normally a lot of mod will switch to neoforge it's the case of a lot of big mod so I think it will be useless to do that and like mentione forge are heavy so if you need to remake the api even if it similare you have a lot of function to do so I don't think he will do that

**Comment by Su5eD** [comment]
This is not planned
